src/com/owncloud/android/ui/adapter/DiskLruImageCache.java

```diff
@@ -120,10 +151,12 @@ public Bitmap getBitmap( String key ) {
             }
             final InputStream in = snapshot.getInputStream( 0 );
             if ( in != null ) {
-                final BufferedInputStream buffIn = 
+                final BufferedInputStream buffIn =
                 new BufferedInputStream( in, IO_BUFFER_SIZE );
-                bitmap = BitmapFactory.decodeStream( buffIn );              
-            }   
+                bitmap = BitmapFactory.decodeStream( buffIn );
+                buffIn.close();
```

Comment:
Nice. Even better, both close() methods could be moved into the ```finally``` block below, with a NULL check condition, so that the are done even in case of Exceptions.